service-wcs/src/main/java/org/oskari/wcs/capabilities/Capabilities.java

```diff
@@ -0,0 +1,116 @@
+package org.oskari.wcs.capabilities;
+
+import java.util.Optional;
+import org.oskari.wcs.extension.Interpolation;
+import org.oskari.wcs.util.WCS;
+
+public class Capabilities {
+
+    private final String updateSequence;
+    private final ServiceIdentification serviceIdentification;
+    private final OperationsMetadata operationsMetadata;
+    private final ServiceMetadata serviceMetadata;
+    private final Contents contents;
+
+    public Capabilities(String updateSequence, ServiceIdentification serviceIdentification,
+            OperationsMetadata operationsMetadata, ServiceMetadata serviceMetadata,
+            Contents contents) {
+        this.updateSequence = updateSequence;
+        this.serviceIdentification = serviceIdentification;
+        this.operationsMetadata = operationsMetadata;
+        this.serviceMetadata = serviceMetadata;
+        this.contents = contents;
+    }
+
+    public String getUpdateSequence() {
+        return updateSequence;
+    }
+
+    public ServiceIdentification getServiceIdentification() {
+        return serviceIdentification;
+    }
+
+    public OperationsMetadata getOperationsMetadata() {
+        return operationsMetadata;
+    }
+
+    public ServiceMetadata getServiceMetadata() {
+        return serviceMetadata;
+    }
+
+    public Contents getContents() {
+        return contents;
+    }
+
+    /**
+     * Check if the coverage appears in the GetCapabilities response
+     */
+    public boolean servesCoverage(String coverageId) {
+        if (coverageId == null || coverageId.length() == 0) {
```

Comment:
String.isEmpty() is used in other places of Oskari. No biggie - just a heads up.